src/plugins/irc/irc-config.c

```diff
@@ -3227,6 +3246,30 @@ irc_config_init ()
         NULL, NULL, NULL,
         &irc_config_change_color_item_nick_modes, NULL, NULL,
         NULL, NULL, NULL);
+    irc_config_color_item_tls_version_max = weechat_config_new_option (
+        irc_config_file, ptr_section,
+        "item_tls_version_max", "color",
+        N_("color for max supported tls version in bar item \"tls_version\""),
+        NULL, -1, 0, "green", NULL, 0,
+        NULL, NULL, NULL,
+        &irc_config_change_color_item_tls_version, NULL, NULL,
+        NULL, NULL, NULL);
+    irc_config_color_item_tls_version_1_2 = weechat_config_new_option (
```

Comment:
I would rename this variable `irc_config_color_item_tls_version_deprecated` and not mention any TLS version, because in future more TLS versions could be marked as deprecated. So we'll not have to rename this option.